src/processor/LakeOfFHIR.ts

```diff
@@ -0,0 +1,73 @@
+import { fhirdefs, utils } from 'fsh-sushi';
+import { WildFHIR } from './WildFHIR';
+
+// Like FSHTank in SUSHI, but it doesn't contain FSH, it contains FHIR.  And who ever heard of a tank of FHIR?  But a lake of FHIR...
+export class LakeOfFHIR implements utils.Fishable {
+  constructor(public readonly docs: WildFHIR[]) {}
```

Comment:
Why the consistent use of `docs` when describing the contents of `LakeOfFhir`? Would something like either `wilds` make more sense to match `WildFHIR`, or something like `resources` make sense? Unless I'm missing something all these docs should be `resources` right?

Reply:
It was kind of inspired by `FSHTank`, which has `docs` -- but `docs` probably makes more sense there than here.  I think I shied away from `resources` because in other places you would expect that to be the FHIR JSON, but this is the `WildFHIR` wrapper (so we can keep a reference to the path).  What do you think?  Do you have a preference for `resources`, `wilds`, or something else?

Reply:
I guess I wasn't thinking about the fact that the resources are wrapped. In that case I think docs is fine, I'm not sure `wilds` is better. That may just be because the word `wilds` sounds weird to me, but although `docs` is somewhat confusing here, it does align well with what we think of as a `doc` in SUSHI, so I'm fine with `docs`.